.flake8

```
@@ -3,9 +3,12 @@ max-complexity = 6
 inline-quotes = double
 max-line-length = 88
 extend-ignore = E203
-docstring_style=sphinx
 
 ignore =
+  ; flake8-rst-docstrings
+  RST,
```

Disabling a whole set of rules seems a bit heavy handed. At https://github.com/peterjc/flake8-rst-docstrings it suggest a subset of rules like
```suggestion
  ; Google Python style is not RST until after processed by Napoleon
  ; See https://github.com/peterjc/flake8-rst-docstrings/issues/17
  RST201,RST203,RST301,
```

---

Ah thanks! I thought this whole set of errors was specific to docstrings formatted in the sphinx-style, thanks for pointing this out.